README.md

```diff
@@ -9,7 +9,7 @@ A Dockerfile for Sonatype Nexus Repository Manager 3, based on CentOS.
 To run, binding the exposed port 8081 to the host.
 
```

Comment:
I do not object but would prefer if someone seconds as it's also the same name as the github repo (which now doesn't match), and since I didn't write this am unsure of the intent.  I am actually not sure the value of that header line at all.